doc/model/train-se-atten.md

```diff
@@ -0,0 +1,109 @@
+* The {ref}`neuron <model/descriptor[se_atten]/neuron>` specifies the size of the embedding net. From left to right the members denote the sizes of each hidden layer from input end to the output end, respectively. If the outer layer is of twice size as the inner layer, then the inner layer is copied and concatenated, then a [ResNet architecture](https://arxiv.org/abs/1512.03385) is built between them.
+* The {ref}`axis_neuron <model/descriptor[se_atten]/axis_neuron>` specifies the size of submatrix of the embedding matrix, the axis matrix as explained in the [DeepPot-SE paper](https://arxiv.org/abs/1805.09003) 
+* If the option {ref}`resnet_dt <model/descriptor[se_atten]/resnet_dt>` is set to `true`, then a timestep is used in the ResNet.
+* {ref}`seed <model/descriptor[se_atten]/seed>` gives the random seed that is used to generate random numbers when initializing the model parameters.
+* {ref}`attn <model/descriptor[se_atten]/attn>` sets the length of hidden vector during scale-dot attention computation.
+* {ref}`attn_layer <model/descriptor[se_atten]/attn_layer>` sets the number of layers in attention mechanism.
+* {ref}`attn_mask <model/descriptor[se_atten]/attn_mask>` determines whether to mask the diagonal in the attention weights and False is recommended.
+* {ref}`attn_dotr <model/descriptor[se_atten]/attn_dotr>` determines whether to dot the relative coordinates on the attention weights as a gated scheme, True is recommended.
+
+## Fitting `"ener"`
+DPA-1 only support `"ener"` fitting type, and you can refer [here](train-energy.md) for detail information.
+
+## Type embedding
+DPA-1 only support models with type embeddings on. And the default setting is as follows:
+```json
+"type_embedding":{
+            "neuron":           [2, 4, 8],
+            "resnet_dt":        false,
+            "seed":             1
+        },
+```
+You can add these settings in input.json if you want to change the defaul ones, see [here](train-se-e2-a-tebd.md) for detail information.
+
+
+## Type map
+For training a large systems, especially those with dozens of elements, the {ref}`type <model/type_map>` determines the element index of training data:
+```json
+"type_map": [
+   "Mg",
+   "Al",
+   "Cu"
+  ]
+```
+which should include all the elements in the dataset you want to train on. The detail of data format can be found in [here](data/data-conv.md).
+Note that this data format requires that, only those frames with the same fingerprint(i.e. the number of atoms of different element) can be put together as a unit system.
+This may lead to sparse frame number in those rare systems. 
+
+An ideal way is to put systems with same total number of atoms together, which is the way we trained DPA-1 on OC2M. This API will be uploaded on dpdata soon for a more convenient experience.
```

Comment:
You should explain, in detail, how one prepares the training data for DPA-1. Someone may want to manually prepare data, rather than using dpdata.